packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -944,6 +954,17 @@ class CachedLocalEngineArtifacts implements LocalEngineArtifacts {
     throw Exception('Unsupported platform $platform.');
   }
 
+  String _flutterTesterFractionalPath(TargetPlatform platform) {
+    if (_platform.isLinux) {
+      return _fileSystem.path.join(engineOutPath, _artifactToFileName(Artifact.flutterTester));
+    } else if (_platform.isMacOS) {
+      return _fileSystem.path.join(engineOutPath, 'flutter_tester_fractional_translation');
```

Comment:
Do you know why this is different from `_artifactToFileName(Artifact.flutterTester)`?

Reply:
Don't know, tried to remove it, lets see what happens